pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -35,11 +35,21 @@ python3.pkgs.buildPythonApplication rec {
     pygobject3 ipaddress libvirt libxml2 requests cdrtools
   ];
 
-  patchPhase = ''
+  prePatch = ''
```

Comment:
This is wrong and breaks patches that touch this file. Use postPatch instead.

Reply:
ah ok. In this case it doesn't touch anything the patch (single line change) touches. But I get your point and will watch out for it in the future :+1: